src/kernel/include/shared/cos_types.h

```diff
@@ -223,40 +223,54 @@ captbl_idsize(cap_t c)
  * LLBooter initial captbl setup:
  * 0 = sret,
  * 1-3 = nil,
- * 4-5 = this captbl,
- * 6-7 = our pgtbl root,
- * 8-11 = our component,
- * 12-13 = vm pte for booter
- * 14-15 = untyped memory pgtbl root,
- * 16-17 = vm pte for physical memory,
- * 18-19 = km pte,
- * 20-21 = comp0 captbl,
- * 22-23 = comp0 pgtbl root,
- * 24-27 = comp0 component,
- * 28~(20+2*NCPU) = per core alpha thd
+ * 4-7 = memory fault handler,
+ * 8-11 = divide by zero fault handler,
+ * 12-15 = breapoint fault handler,
```

Comment:
please enable spell checking in your editor.  I don't want to be a spellchecker when a machine should be doing that.